package.json

```diff
@@ -151,7 +151,7 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-spec-reporter": "^0.0.31",
     "karma-webpack": "^5.0.0",
-    "lerna": "^4.0.0",
+    "lerna": "^6.6.2",
```

Comment:
This is going to cause lots of headache 🚨 Sorry about that.

Reply:
Yep, I tried to use the oldest with support for provenance... and I don't know if we are ready to support this